packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -35,13 +35,6 @@ export const messages = {
   force_lightning_lwc_stop_not_running:
     'The local development server is not running.',
   force_lightning_lwc_stop_in_progress: 'Stopping local development server',
-  force_lightning_lwc_preview_text: 'SFDX: Preview Component Locally',
-  force_lightning_lwc_preview_file_undefined:
-    "Can't find the Lightning Web Components module. Check that %s is the correct file path.",
-  force_lightning_lwc_preview_file_nonexist:
-    "Can't find the Lightning Web Components module in %s. Check that the module exists.",
-  force_lightning_lwc_preview_unsupported:
-    "Something's not right with the filepath. The local development server doesn't recognize the Lightning Web Components module '%s.'",
```

Comment:
gathered them all together further below, and also changed them all to start with `force_lightning_lwc_preview_...`